src/osemgrep/core/Metrics_.ml

```diff
@@ -279,7 +279,10 @@ let add_engine_type (engine_type : Engine_type.t) =
           secrets_config;
           code_config;
           supply_chain_config;
+          git_remote;
```

Comment:
I don't think this should be in Engine_type; perhaps `Deep_scan_config.t`? Or a field which is generally `None` in `Core_scan_config.t`.

Reply:
I agree. This whole thing feels weird.

Reply:
Ok, changing it to use hooks and have the URL be a project root as Martin recommended